Resources/Locale/en-US/game-ticking/game-ticker.ftl

```diff
@@ -7,7 +7,7 @@ game-ticker-unknown-role = Unknown
 game-ticker-delay-start = Round start has been delayed for {$seconds} seconds.
 game-ticker-pause-start = Round start has been paused.
 game-ticker-pause-start-resumed = Round start countdown is now resumed.
-game-ticker-player-join-game-message = Welcome to Space Station 14! If this is your first time playing, be sure to read the game rules, and don't be afraid to ask for help in LOOC (local OOC) or OOC (usually available only between rounds).
+game-ticker-player-join-game-message = Welcome to Cosmatic Drift! If this is your first time playing, be sure to read the game rules, and don't be afraid to ask for help in LOOC (local OOC) or OOC (remember that talking about IC events in OOC is frowned upon). Don't feel afraid to ahelp! The admins are happy to modify your character, give you an antag role, and do many other things. The worst they can say is no.
```

Comment:
Bit wordy, I'd cut down on the "(remember that talking about IC events in OOC is frowned upon)" part since that's covered in the rules ~~but sadly not common sense~~.

Reply:
The reason why I included it is because there is always a chance we could get a brand new player that asks something innocent over OOC by mistake then gets blasted by 10 people telling them "ick ock" (A term that they might not even know the definition of). Realistically, 90% of players don't read the rules past the first few sentences.

Reply:
~~IF ONLY PEOPLE JUST FUCKING LEFT IT TO ADMINS TO HANDLE INSTEAD OF DOGPILING-~~ ... Yeah sounds good.